tx-backend/src/main/java/org/eclipse/tractusx/traceability/common/model/SecurityUtils.java

```diff
@@ -0,0 +1,40 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+package org.eclipse.tractusx.traceability.common.model;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SecurityUtils {
+
+    public static String sanitize(String unSanitizedInput) {
+        return unSanitizedInput.replaceAll("\r\n|\r|\n", " ");
+    }
+
+    public static List<String> sanitizeList(List<String> unSanitizedList) {
```

Comment:
```suggestion
    public static List<String> sanitize(List<String> unSanitizedList) {
```
I think it would be better to just overload the method with new parameters instead of renaming it. What is your take on this?